diff --git a/CMakeLists.txt b/CMakeLists.txt
index a83ac43..74fc031 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -62,7 +62,7 @@ if(DEPENDENCY_CHECK AND BUILD_POSTGRESQL_CONNECTOR)
 endif()

 if(DEPENDENCY_CHECK AND BUILD_SQLITE3_CONNECTOR)
-  find_package(SQLite3 REQUIRED)
+  find_package(unofficial-sqlite3 REQUIRED)
 endif()

 if(DEPENDENCY_CHECK AND BUILD_SQLCIPHER_CONNECTOR)
@@ -89,7 +89,7 @@ target_compile_features(sqlpp11 INTERFACE cxx_std_11)


 if(BUILD_SQLITE3_CONNECTOR)
-  add_component(NAME sqlite3 DEPENDENCIES SQLite::SQLite3)
+  add_component(NAME sqlite3 DEPENDENCIES unofficial::sqlite3::sqlite3)
 endif()

 if(BUILD_SQLCIPHER_CONNECTOR)

