From a637f28a31bb45835de2e46f02f38c6b82e9e851 Mon Sep 17 00:00:00 2001
From: uuiid <957714080@qq.com>
Date: Fri, 30 Jan 2026 18:24:06 +0800
Subject: [PATCH 2/2] fix-install-dir

---
 dnacalib/CMakeModulesExtra/install/InstallLibrary.cmake | 2 --
 1 file changed, 2 deletions(-)

diff --git a/dnacalib/CMakeModulesExtra/install/InstallLibrary.cmake b/dnacalib/CMakeModulesExtra/install/InstallLibrary.cmake
index d0659f8..b3c4e5d 100644
--- a/dnacalib/CMakeModulesExtra/install/InstallLibrary.cmake
+++ b/dnacalib/CMakeModulesExtra/install/InstallLibrary.cmake
@@ -31,10 +31,8 @@ macro(install_library target_name)
         EXPORT ${PROJECT_NAME}-targets
         COMPONENT ${COMPONENT_NAME}
         RUNTIME
-            DESTINATION ${CMAKE_INSTALL_LIBDIR}
             COMPONENT ${COMPONENT_NAME}
         LIBRARY
-            DESTINATION ${CMAKE_INSTALL_LIBDIR}
             COMPONENT ${COMPONENT_NAME}
             NAMELINK_COMPONENT ${COMPONENT_NAME}
         ARCHIVE
-- 
2.51.1.windows.1

